### PR TITLE
[Relay][Frontend][TFLite] Add parser support for squared difference

### PR DESCRIPTION
**Pull request by wyc-ruiker**
Thanks for contributing to TVM!   Please refer to guideline https://docs.tvm.ai/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/apache/incubator-tvm/blob/master/CONTRIBUTORS.md#reviewers) by @ them in the pull request thread.

### CODE REVIEW

**Comment by wyc-ruiker** [comment]
Could you check this PR, @kevinthesun @FrozenGene

File: tests/python/frontend/tflite/test_forward.py [hunk 1]
```diff
@@ -723,6 +723,14 @@ def _test_greater(data):
     """ One iteration of greater """
     return _test_elemwise(math_ops.greater, data)
 
+#######################################################################
+# Squared_difference
+# -------
```
Comment (FrozenGene) [inline tests/python/frontend/tflite/test_forward.py]: Align - the same length of squared_difference

File: python/tvm/relay/frontend/tflite.py [hunk 2]
```diff
@@ -628,6 +629,15 @@ def convert_greater(self, op):
                 'TFlite quantized greater operator is not supported yet.')
         return self._convert_elemwise(_op.greater, op)
 
+    def convert_squared_difference(self, op):
+        # Check if the input tensor is quantized, call QNN op
+        if self.is_quantized(op):
+            raise tvm.error.OpNotImplemented(
+                'TFlite quantized greater operator is not supported yet.')
```
Comment (FrozenGene) [inline python/tvm/relay/frontend/tflite.py]: comment is wrong. Not greater op.

File: python/tvm/relay/frontend/tflite.py [hunk 3]
```diff
@@ -628,6 +629,15 @@ def convert_greater(self, op):
...
+        difference = self._convert_elemwise(_op.subtract, op)
+        out = _op.multiply(difference, difference)
```
Comment (FrozenGene) [inline python/tvm/relay/frontend/tflite.py]: maybe could consider using `power`.

File: tests/python/frontend/tflite/test_forward.py (hunk 1 above)
Comment (wyc-ruiker) [reply tests/python/frontend/tflite/test_forward.py]: fixed

File: python/tvm/relay/frontend/tflite.py (hunk 2 above)
Comment (wyc-ruiker) [reply python/tvm/relay/frontend/tflite.py]: fixed

File: python/tvm/relay/frontend/tflite.py (hunk 3 above)
Comment (wyc-ruiker) [reply python/tvm/relay/frontend/tflite.py]: fixed

File: python/tvm/relay/frontend/tflite.py [hunk 4]
```diff
@@ -726,6 +727,15 @@ def convert_greater(self, op):
                 'TFlite quantized greater operator is not supported yet.')
         return self._convert_elemwise(_op.greater, op)
 
+    def convert_squared_difference(self, op):
+        # Check if the input tensor is quantized, call QNN op
+        if self.is_quantized(op):
+            raise tvm.error.OpNotImplemented(
+                'TFlite quantized squared difference operator is not supported yet.')
+        difference = self._convert_elemwise(_op.subtract, op)
+        out = _op.power(difference, relay.const(2, 'float32'))
```
[the remark this answers is not in the document]
Comment (wyc-ruiker) [reply python/tvm/relay/frontend/tflite.py]: Thanks for your suggestion. I think `_op.power` is more appropriate here and I have already fixed it.

File: python/tvm/relay/frontend/tflite.py [hunk 5]
```diff
@@ -735,6 +736,16 @@ def convert_greater(self, op):
                 'TFlite quantized greater operator is not supported yet.')
         return self._convert_elemwise(_op.greater, op)
 
+    def convert_squared_difference(self, op):
+        # Check if the input tensor is quantized, call QNN op
+        if self.is_quantized(op):
+            raise tvm.error.OpNotImplemented(
+                'TFlite quantized squared difference operator is not supported yet.')
+        difference = self._convert_elemwise(_op.subtract, op)
+        exp_type = self.get_tensor_type_str(self.get_output_tensors(op)[0].tensor.Type())
```
Comment (FrozenGene) [inline python/tvm/relay/frontend/tflite.py]: better add one comment
``_convert_elemwise has guaranteed only have one output tensor``

Comment (wyc-ruiker) [reply python/tvm/relay/frontend/tflite.py]: done.

**Comment by FrozenGene** [comment]
Thanks @wyc-ruiker LGTM now. Let us wait CI's result.

**Comment by FrozenGene** [comment]
Thanks @wyc-ruiker This is merged.

**Pull request merged**